CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
 #### :rocket: New Feature
 - Variants: Allow coercing from variant to variant, where applicable. https://github.com/rescript-lang/rescript-compiler/pull/6314
 
+#### :boom: Breaking Change
+- Fixed Js.Json.kind type shadowing Js.Json.t type: `type kind<_>` -> `type Kind.kind<_>` https://github.com/rescript-lang/rescript-compiler/pull/6317
```

Comment:
This should say what the breaking change is and what is the fix for someone relying on the previous behaviour.

Reply:
https://github.com/rescript-lang/rescript-compiler/pull/6317/commits/a02f005ec588eee0c652abcf1e527f5823d058c1